langserver/langserver.py

```diff
@@ -99,10 +101,11 @@ def workspace_modules(self, path) -> List[Module]:
     def workspace_symbols(self):
         if self.symbol_cache:
             return self.symbol_cache
-        symbols = []
         py_paths = (path for path in self.fs.walk(self.root_path) if path.endswith(".py"))
-        for path, src in self.fs.batch_open(py_paths):
-            symbols.extend(extract_symbols(src, path))
+        py_srces = self.fs.batch_open(py_paths)
+        with multiprocessing.Pool() as p:
+            symbols_chunks = p.imap_unordered(extract_symbols_star, py_srces)
```

Comment:
Is there overhead associated with this over regular `map`? Documentation says "For very long iterables using a large value for chunksize can make the job complete much faster than using the default value of 1." Should we set a threshold at which the chunksize is increased?